src/sbvr-api/errors.ts

```diff
@@ -1,5 +1,14 @@
 import TypedError = require('typed-error');
 
+export class PayloadError extends TypedError {
+	constructor(
+		name: string,
```

Comment:
This will end up in the `message` property right?
Is there a specific that this is the first arguments and not optional?